functions/test/endpoints/PointLog-Tests/Handle.test.ts

```diff
@@ -50,39 +51,36 @@ describe('point_log/handle', async ()  => {
         await FirestoreDataFactory.setUser(db, PRIV_RES, 4)
         await FirestoreDataFactory.setUser(db, EA_ID, 5)
         await FirestoreDataFactory.setPointType(db, 1)
-        await FirestoreDataFactory.setPointType(db, 2, {residents_can_submit: false})
-        await FirestoreDataFactory.setPointType(db, 3, {is_enabled:false})
+        await FirestoreDataFactory.setPointType(db, 1, {description: PT_1_DETAILS, name: PT_1_DETAILS, value: 10})
+        await FirestoreDataFactory.setPointType(db, 2, {residents_can_submit: false, value: 10})
+        await FirestoreDataFactory.setPointType(db, 3, {is_enabled:false, value: 10})
         await FirestoreDataFactory.setHouse(db, HOUSE_NAME)
-        await FirestoreDataFactory.setHouseCode(db, HOUSE_CODE)
+
+        await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, {id: PARAM_CHECKS_LOG_ID})
+        await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, true, {id: REJECT_LOG_ID, description: REJECT_LOG_ID, point_type_id: 1, approved: false})
     })
 
     beforeEach(async () => {
-        let approved_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, POINT_LOG_DEFAULTS)
+        let approved_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false)
         if (approved_log_ref !== null) {
             approved_log = (approved_log_ref as firebase.firestore.DocumentReference)
-            point_value = 1
             point_description = POINT_LOG_DEFAULTS.description!
             approved_log = approved_log.id
         }
         const approve_body = {"approve":true, "point_log_id":approved_log}
         await factory.post(point_log_func, HANDLE_POINT_PATH, approve_body, RHP_ID)
 
-        let rejected_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, POINT_LOG_DEFAULTS)
+        let rejected_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false)
         if (rejected_log_ref !== null) {
             rejected_log = (rejected_log_ref as firebase.firestore.DocumentReference).id
         }
         const rejected_body = {"approve":false, "point_log_id":rejected_log}
         await factory.post(point_log_func, HANDLE_POINT_PATH, rejected_body, RHP_ID)
 
-        let unhandled_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, POINT_LOG_DEFAULTS)
+        let unhandled_log_ref = await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false)
         if (unhandled_log_ref !== null) {
             unhandled_log = (unhandled_log_ref as firebase.firestore.DocumentReference).id
```

Comment:
You should not use the endpoint you are testing in the setup of your tests. This makes it hard to distinguish what is causing an error when something fails.

All of your data should be set up once in the beforeAll function and each method should use a different and independent set of data. This helps make sure that each test is independent of the others.

If some of the features you need to make data are not present in the FirestoreDataFactory, feel free to add the necessary features so you can do all of your test set up.